pg-index-health-testing/src/main/java/io/github/mfvanek/pg/testing/PostgreSqlClusterWrapper.java

```diff
@@ -159,4 +180,33 @@ private void throwErrorIfNotInitialized() {
             throw new AssertionError("not initialized");
         }
     }
+
+    /**
+     * Provide convenient way to create cluster with single username/password.
+     * If no username/password is specified, "customuser" and "custompassword" will be used as default values for username and password, respectively
+     */
+    public static class Builder {
+
+        private String username;
+        private String password;
+
+        public Builder() {
```

Comment:
The main style in the project is that all builders have private constructors

Reply:
I saw a static method that instantiates the builder. What is the point of doing it this way?

Reply:
It's a common and a widely used convention